Lib/traceback.py

```diff
@@ -311,11 +311,11 @@ class FrameSummary:
     """
 
     __slots__ = ('filename', 'lineno', 'end_lineno', 'colno', 'end_colno',
-                 'name', '_lines', '_lines_dedented', 'locals')
+                 'name', '_lines', '_lines_dedented', 'locals', '_code')
 
     def __init__(self, filename, lineno, name, *, lookup_line=True,
             locals=None, line=None,
-            end_lineno=None, colno=None, end_colno=None):
+            end_lineno=None, colno=None, end_colno=None, **kwargs):
```

Comment:
Is there a particular reason to use `kwargs` instead of `_code` or `code` here? This is a public interface but the documentation is already out of date (should we fix it some time? Not in this PR of course). Feels like `code=None` would be better here.